app/actualize_script.php

```diff
@@ -12,8 +12,8 @@
 // Set the header params ($_GET) to call the FRSS application.
 $_GET['c'] = 'feed';
 $_GET['a'] = 'actualize';
-$_GET['ajax'] = 1;
 $_GET['maxFeeds'] = PHP_INT_MAX;
+$_SERVER['HTTP_X_REQUESTED_WITH'] = 'XMLHttpRequest';
```

Comment:
```suggestion
$_SERVER['HTTP_X_AJAX'] = '1';
```

I am guessing this has been forgotten from previous changes

Reply:
Indeed